ui/src/components/ui/device-control-panel/device-control-card/device-control-card.tsx

```diff
@@ -0,0 +1,96 @@
+import { useState } from 'react'
+import { Button, Card, Div, Flex, Headline, Tooltip } from '@vkontakte/vkui'
+import {
+  Icon24Upload,
+  Icon20BugOutline,
+  Icon20HelpOutline,
+  Icon20CopyOutline,
+  Icon20GlobeOutline,
+  Icon20DeleteOutline,
+  Icon28SettingsOutline,
+  Icon20AddSquareOutline,
+  Icon28StopwatchOutline,
+  Icon20ChevronRightOutline,
+} from '@vkontakte/icons'
+
+import { ConditionalRender } from '@/components/lib/conditional-render'
+
+import { DeviceControlCardAfter, DeviceControlCardIcon } from './types'
+
+import styles from './device-control-card.module.css'
+
+import type { ReactNode } from 'react'
+
+const BEFORE_ICON_MAP = {
+  [DeviceControlCardIcon.SETTINGS]: <Icon28SettingsOutline height={20} width={20} />,
+  [DeviceControlCardIcon.UPLOAD]: <Icon24Upload height={20} width={20} />,
+  [DeviceControlCardIcon.BUG]: <Icon20BugOutline />,
+  [DeviceControlCardIcon.GLOBE]: <Icon20GlobeOutline />,
+  [DeviceControlCardIcon.COPY]: <Icon20CopyOutline />,
+  [DeviceControlCardIcon.SHELL]: <Icon20ChevronRightOutline />,
+  [DeviceControlCardIcon.TIME]: <Icon28StopwatchOutline height={20} width={20} />,
+}
+
+const AFTER_BUTTON_ICON_MAP = {
+  [DeviceControlCardAfter.ADD]: <Icon20AddSquareOutline />,
+  [DeviceControlCardAfter.COPY]: <Icon20CopyOutline />,
+  [DeviceControlCardAfter.CLEAR]: <Icon20DeleteOutline />,
+}
+
+type DeviceControlCardProps = {
+  title: string
+  children: ReactNode
+  icon?: DeviceControlCardIcon
```

Comment:
Maybe make this `after` and allow for arbitrary content and not just icons?
And pass icons as DeviceControlCardIcon in both `after` and `before`?